### PR TITLE
added login screen

### PR DESCRIPTION
**Pull request by Pelozo**
__Summary__: Added log-in screen from card U1

__Screenshots__: [original design](https://i.imgur.com/O4WEFzq.jpg) | [implementation](https://i.imgur.com/9WGm6kX.png)

__Known Issues/Notes__: Had to disable WARP (doesn't work in windows)

__Trello Card__: https://trello.com/c/FHt3EAis

### CODE REVIEW

File: WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            view?.showErrorAllRequired()
+            return
+        }
+        if (!user.isEmailValid()) {
+            view?.showErrorInvalidEmail()
+            return
+        }
+        userSession.username = user
+        view?.goToHome()
+    }
```
Comment (ffasitella) [inline WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.kt]: You can use `view?.apply`

File: WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/utils/ExtensionFunctions.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package ar.com.wolox.android.example.utils
```
Comment (ffasitella) [inline WoloxAndroidBootstrap/app/src/main/java/ar/com/wolox/android/example/utils/ExtensionFunctions.kt]: We should group extension functions by the class they extend. For instance, every String extension function goes to `StringExtensionFunctions.kt`

File: WoloxAndroidBootstrap/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -12,9 +12,19 @@
...
 
+
+    <string name="fragment_login_email">Email</string>
+    <string name="fragment_login_password">Password</string>
+    <string name="fragment_login_sign_up">Sign up</string>
+    <string name="fragment_login_terms_and_cond">By joining you agree to our Terms &amp; conditions</string>
+    <string name="fragment_login_error_all_fields_required">All fields are required</string>
+    <string name="fragment_login_error_invalid_email">Invalid email format</string>
+    <string name="fragment_signup_join">Join</string>
+    <string name="fragment_signup_terms_and_cond">By joining you agree to our <u>Terms &amp; conditions</u></string>
```
Comment (ffasitella) [inline WoloxAndroidBootstrap/app/src/main/res/values/strings.xml]: Use this wording in the login screen, it has the underlined text!

**Review by ffasitella — CHANGES_REQUESTED** [review]
I left some small suggestions!
Excellent job 👍🏽

**Review by jcruz-wolox — APPROVED** [review]
Approved for met, good job Leo!

**Review by ffasitella — APPROVED** [review]
Amazing job 🚀 ✨

**Pull request merged**